core/src/contracts.ts

```diff
@@ -12,6 +12,11 @@ export const getClipperNameByCollateralType = function (collateralType: string):
     return `MCD_CLIP_${suffix}`;
 };
 
+export const getJoinNameByCollateralType = function (collateralType: string): string {
```

Comment:
q: what's the join name of a collateral type?

Reply:
It's a helper to construct contract name `MCD_JOIN_*`. I guess it will be better to call it `getJoinContractNameByCollateralType`, but since it's already approved I will change it in another PR